manifests/n/NHNCorporation/Dooray!Messenger/1.1.5.1/NHNCorporation.Dooray!Messenger.locale.ko-KR.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: NHNCorporation.Dooray!Messenger
+PackageVersion: 1.1.5.1
+PackageLocale: ko-KR

```

```suggestion
PackageLocale: en-US
```
This is not 🤦‍♂️
Remember to change the name :))